src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1034,4 +1034,22 @@ This version used in: {2}</value>
   <data name="ConvertToInterpolatedString" xml:space="preserve">
     <value>Convert to interpolated string</value>
   </data>
+  <data name="MakePartialDefinitionForType" xml:space="preserve">
+    <value>Make Partial Definition For {0}</value>
+  </data>
+  <data name="MakePartialDefinitionForTypeInNewFile" xml:space="preserve">
+    <value>Make Partial Definition For {0}...</value>
+  </data>
+  <data name="MoveTypeToFileName" xml:space="preserve">
+    <value>Move Type To {0}</value>
+  </data>
+  <data name="MoveTypeToNewFile" xml:space="preserve">
+    <value>Move {0}...</value>
+  </data>
+  <data name="RenameFileToMatchTypeName" xml:space="preserve">
+    <value>Rename File To {0}</value>
+  </data>
+  <data name="RenameTypeToMatchFileName" xml:space="preserve">
```

Comment:
Loc names should match english string content.  That way we can tell in source what this loc string will actually look like to the user.